src/entities/Proposal/outcomeUtils.ts

```diff
@@ -22,7 +22,7 @@ export function calculateWinnerChoice(result: Scores) {
       }
       return winner
     })
-  const winnerVotingPower = result[winnerChoice]
+  const winnerVotingPower = Math.round(result[winnerChoice])
```

Comment:
why do you need to round here? I think we should try to use the exact numbers for calculating winner choice

Reply:
We are rounding the final result